esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/workflow.rst

```diff
@@ -27,8 +27,8 @@ The |RTW| performs the following steps:
   :Executes:
      The ``configure.py`` script from the |Rose| app
   :Details:
-     Runs once at the start of the workflow, immediately after the successful
-     completion of the ``install_cold`` job
+     Runs once at the start of each cycle, immediately after the successful
+     completion of the previous cycle's compare job
```

Comment:
Could you also add that it runs at the beginning after install cold please?

Reply:
https://github.com/ESMValGroup/ESMValTool/pull/2822/commits/db58e6ff82988db6a05347c24136c32d46841273